src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -526,7 +496,6 @@ export function useUsersValidation(): UsersStepValidation {
   }
 
   for (let index = 0; index < users.length; index++) {
-    const invalidGroups = [];
```

Comment:
I see the problem, but leaving the groups code in doesn't seem to cause any issues with the tests. Can we leave the code in to keep at least some groups field validation and follow up by fixing the issue you've opened?